packages/core/src/application.ts

```diff
@@ -121,15 +135,18 @@ export class Application extends Context {
     request: ServerRequest,
     response: ServerResponse,
   ) {
-    if (request.method === 'GET' && request.url === '/openapi.json') {
+    if (request.method === 'GET' && request.url &&
+      request.url in OPENAPI_SPEC_MAPPING) {
```

Comment:
The formatting is different from what we prescribe in our style guide, see http://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if

> Still acceptable:

```js
if (testInEquality({gte: example.between[0]}, value) &&
    testInEquality({lte: example.between[1]}, value) &&
    testInEquality({lte: example.between[2]}, value)) {
  handleInEquality();
}
```

> **Bad:**
>
> One level of indentation makes it difficult to tell the difference between the condition and the branch body.

```js
if (testInEquality({gte: example.between[0]}, value) &&
  testInEquality({lte: example.between[1]}, value) &&
  testInEquality({lte: example.between[2]}, value)) {
  handleInEquality();
}
```

Is this formatting the outcome of prettier?